modules/diskq/logqueue-disk-reliable.c

```diff
@@ -86,7 +86,7 @@ _ack_backlog(LogQueue *s, gint num_msg_to_ack)
   for (i = 0; i < num_msg_to_ack; i++)
     {
       gint64 pos;
-      if (qdisk_get_backlog_head (self->super.qdisk) == qdisk_get_reader_head (self->super.qdisk))
+      if (qdisk_get_backlog_head(self->super.qdisk) == qdisk_get_head_position(self->super.qdisk))
```

Comment:
`qdisk_get_reader_head` and the new function `qdisk_get_head_position` looks the same. Can you just rename the old function, please? It will be easier later to follow thorugh the git history.

Reply:
Actually I do not think, that `qdisk_get_reader_head()` should even be a thing. I would not rename it, but maybe I can remove it at this point, and change all its other users.

Reply:
As discussed in https://github.com/syslog-ng/syslog-ng/pull/3752#discussion_r688478400 2 interfaces will co-exist fow now.